crates/bevy_render/src/lib.rs

```diff
@@ -216,10 +216,54 @@ pub enum RenderSystems {
     PostCleanup,
 }
 
+/// The schedule that contains the app logic that is evaluated each tick
+///
+/// This is highly inspired by [`bevy_app::Main`]
+#[derive(ScheduleLabel, Clone, Debug, PartialEq, Eq, Hash, Default)]
+pub struct MainRender;
+impl MainRender {
+    pub fn run(world: &mut World, mut run_at_least_once: Local<bool>) {
+        if !*run_at_least_once {
+            world.resource_scope(|world, order: Mut<MainRenderScheduleOrder>| {
+                for &label in &order.startup_labels {
+                    let _ = world.try_run_schedule(label);
```

Comment:
i presume this isnt discarding errors in a way that doesnt print them, right? i would not like it if stuff failed silently

Reply:
That's how the schedule for the main world does thing, it was literally copy pasted, so I assumed it's correct.